docs/en/sql-reference/sql-statements/loading_unloading/INSERT.md

```diff
@@ -33,20 +34,38 @@ You can submit an asynchronous INSERT task using [SUBMIT TASK](ETL/SUBMIT_TASK.m
+| column_name   | The name of the destination column(s) to load data in. It must be set as columns that exist in the destination table. <ul><li>If the property `match_column_by` is not set or set to `position` (default), the destination columns you specify are mapped one on one in sequence to the columns of the source table, regardless of what the destination column names are. </li><li>If `match_column_by` is set to `name`, the destination columns are mapped to columns with the same names in the source table, regardless of the column orders in destination and source tables.</li><li>If no destination column is specified, the default value is all columns in the destination table.</li><li>If the specified column in the source table does not exist in the destination column, the default value will be written to this column. </li><li>If the specified column does not have a default value, the transaction will fail.</li><li>If the column type of the source table is inconsistent with that of the destination table, the system will perform an implicit conversion on the mismatched column.</li><li>If the conversion fails, a syntax parsing error will be returned.</li></ul>**NOTE**<br />From v3.3.1, specifying a column list in the INSERT INTO statement on a Primary Key table will perform Partial Updates (instead of Full Upsert in earlier versions). If the column list is not specified, the system will perform Full Upsert. |
+| PROPERTIES    | Properties of the INSERT job. Each property must be a key-value pair. For supported properties, see [PROPERTIES](#properties). |
 | expression    | Expression that assigns values to the column.                |
 | DEFAULT       | Assigns default value to the column.                         |
 | query         | Query statement whose result will be loaded into the destination table. It can be any SQL statement supported by StarRocks. |
 | FILES()       | Table function [FILES()](../../sql-functions/table-functions/files.md). You can use this function to unload data into remote storage. |
 
+### PROPERTIES
+
+INSERT statements support configuring PROPERTIES from v3.4.0 onwards.
+
+| Property         | Description                                                  |
+| ---------------- | ------------------------------------------------------------ |
+| timeout          | The timeout duration of the INSERT job. Unit: Seconds. You can also set the timeout duration for INSERT within the session or globally using the variable `insert_timeout`. |
```

Comment:
What's the order of this parameters?

Reply:
usually in the sequence of the time when they are supported